### PR TITLE
Update licensing-model-azure-hybrid-benefit-ahb-change.md

### PR DESCRIPTION
**Pull request by ujpat**
Replaced by https://github.com/MicrosoftDocs/sql-docs-pr/pull/28277.

### CODE REVIEW

File: azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md [hunk 1]
```diff
@@ -142,12 +142,15 @@ This error occurs when you try to change the license model on a SQL Server VM th
 
 You'll need to [register your SQL Server VM with the SQL IaaS Agent extension](sql-agent-extension-manually-register-single-vm.md).
 
-**Change licensing to AHB, BYOL, HADR or PAYG**
+**Change licensing to AHB, HADR or PAYG**
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md]:
```suggestion
**Change licensing to AHB, HADR, or PAYG**
```

File: azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md [hunk 2]
```diff
@@ -142,12 +142,15 @@ This error occurs when you try to change the license model on a SQL Server VM th
...
 
 Make sure your [subscription is registered with resource provider (RP)](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-subscription-with-rp).
 
 The SQL IaaS Agent extension is required to change the license. Make sure you [remove and reinstall the SQL IaaS Agent extension](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-in-full-mode) if it's in a failed state.
 
+**How Can I deploy SQL Server BYOL(Bring your own License) Image?**
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md]:
```suggestion
**How Can I deploy a SQL Server BYOL (bring your own license) image?**
```

File: azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md [hunk 3]
```diff
@@ -142,12 +142,15 @@ This error occurs when you try to change the license model on a SQL Server VM th
...
+BYOL images have been retired from Azure Marketplace. You can deploy a pay-as-you-go (PAYG) gallery image and change the license to (BYOL) by enabling the [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/)
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md]:
```suggestion
BYOL images have been retired from Azure Marketplace. You can deploy a pay-as-you-go (PAYG) gallery image and change the license to (BYOL) by enabling the [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/).
```

**Review by rwestMSFT — CHANGES_REQUESTED** [review]

File: azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md [hunk 4]
```diff
@@ -142,12 +142,15 @@ This error occurs when you try to change the license model on a SQL Server VM th
...
 You'll need to [register your SQL Server VM with the SQL IaaS Agent extension](sql-agent-extension-manually-register-single-vm.md).
 
-**Change licensing to AHB, BYOL, HADR or PAYG**
+**Change licensing to AHB, HADR, or PAYG**
 
 Make sure your [subscription is registered with resource provider (RP)](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-subscription-with-rp).
 
 The SQL IaaS Agent extension is required to change the license. Make sure you [remove and reinstall the SQL IaaS Agent extension](/azure/azure-sql/virtual-machines/windows/sql-agent-extension-manually-register-single-vm#register-in-full-mode) if it's in a failed state.
 
+**How Can I deploy a SQL Server BYOL (bring your own license) image?**
```
Comment (rwestMSFT) [inline azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md:151]:
```suggestion
**How Can I deploy a SQL Server BYOL (bring your own license) image?**

```

**Comment by Jak-MS** [comment]
@bluefooted 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md [hunk 5]
```diff
@@ -142,12 +142,16 @@ This error occurs when you try to change the license model on a SQL Server VM th
...
+
+BYOL images have been retired from Azure Marketplace. You can deploy a pay-as-you-go (PAYG) gallery image and change the license to (BYOL) by enabling the [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/).
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/licensing-model-azure-hybrid-benefit-ahb-change.md]:
```suggestion
BYOL images have been retired from Azure Marketplace. You can choose a SQL Server Standard or Enterprise edition marketplace image, and enable [Azure Hybrid Benefit](https://azure.microsoft.com/pricing/hybrid-benefit/faq/) during deployment, to create a SQL virtual machine using an existing license.
```

**Review by bluefooted — CHANGES_REQUESTED** [review]

**Comment by rwestMSFT** [comment]
Replaced by https://github.com/MicrosoftDocs/sql-docs-pr/pull/28277.

**Pull request closed without merging**